specification/configuration.md

```diff
@@ -135,7 +135,11 @@ beyond the OpenTelemetry specification exist.
 - `OTEL_TRACES_EXPORTER`
   - Non-RUM distribution MUST default to `otlp` over gRPC with an endpoint of `localhost:4317`
   - Non-RUM distribution MUST offer `jaeger-thrift-splunk` that defaults to `http://127.0.0.1:9080/v1/trace`
-
+- `OTEL_LOG_LEVEL`
+  - Distribution SHOULD support `DEBUG` setting. With this value all relevant troubleshooting data should be logged. 
+  - Distribution MAY support additional levels:
+    - `INFO` - coarser-grained informational events than the `DEBUG`.
+    - `TRACE` - finer-grained informational events than the `DEBUG`.
```

Comment:
If we're going to use `OTEL_LOG_LEVEL` (which is a good idea, since it's stable in the otel spec) I believe we should wait for the OTel spec issues to get resolved before deciding on anything here.